lang/cs/Org.Apache.REEF.Common/Tasks/TaskConfiguration.cs

```diff
@@ -30,11 +26,8 @@
 
 namespace Org.Apache.REEF.Common.Tasks
 {
-    public class TaskConfiguration : ConfigurationModuleBuilder
+    public sealed class TaskConfiguration : ConfigurationModuleBuilder
```

Comment:
If you keep the original TaskConfiguration, I would suggest to change this one as TaskConfigurationModule. We should always have a separate class for configuration modules.